assets/web/header.html

```diff
@@ -5,6 +5,8 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Sunshine</title>
+    <link rel="icon" type="image/x-icon" href="/images/favicon.ico">
```

Comment:
favicon is showing all sizes... need to replace with single size .ico file.

![image](https://user-images.githubusercontent.com/42013603/153116915-932ce858-1b6e-42b9-8fd6-9ed24fb5e766.png)

Reply:
Fixed